bitrise.yml

```diff
@@ -3,8 +3,6 @@ format_version: '8'
 default_step_lib_source: 'https://github.com/bitrise-io/bitrise-steplib.git'
 project_type: ios
 trigger_map:
-- push_branch: '*'
-  workflow: frames-test-pods
```

Comment:
Unrelated change, but it was in the same file. Stopping the workflows from taking place when just making a push to remote.